### PR TITLE
PSS (jobs), the deployment itself is ok

### PR DESCRIPTION
**Pull request by jkremser**
with this change the only issue is `EndpointSlice` (control plane thing)

```
kubectl get polr -A -n giantswarm -ojsonpath='{.items[*].results[?(@.result=="fail")]}' | jq 'select(.resources[].name | contains("caip")) | .message'                                 ○ gcapeverde-admin@gcapeverde
"discovery.k8s.io/v1beta1/EndpointSlice is deprecated and will be removed in v1.25. See: https://kubernetes.io/docs/reference/using-api/deprecation-guide/"
"discovery.k8s.io/v1beta1/EndpointSlice is deprecated and will be removed in v1.25. See: https://kubernetes.io/docs/reference/using-api/deprecation-guide/"
```

### CODE REVIEW

**Review by erkanerol — APPROVED** [review]

**Comment by erkanerol** [comment]
@jkremser can you note the EndpointSlice issue in somewhere in our board?

**Comment by jkremser** [comment]
> can you note the EndpointSlice issue in somewhere in our board?

It's not a big deal, it will go away when we upgrade to 1.25, I put a comment here about it: https://github.com/giantswarm/cluster-service/pull/582#issuecomment-1740773347

tl;dr v1beta1 -> v1 upgrade/graduation
